actionpack/test/controller/mime/accept_format_test.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "abstract_unit"
+ActionView::Base.default_formats ||= Mime::SET.symbols
```

Comment:
@Edouard-chin just double checking, is there another way we should express this? Or make it obsolete another way? From the issue it seems like this was more fallout from https://github.com/rails/rails/pull/38026 😊

Reply:
👋 Thanks for investigating the issue @yahonda ! I need to dig a bit deeper to understand the issue. I don't understand yet why this test ever worked since https://github.com/rails/rails/blob/22483b86a6c779743b30e2f23bb46accfbf96b28/actionpack/lib/action_dispatch.rb#L117 shouldn't have been evaluated as AFAIK we don't run railtie in actionpack test suite.  Will post back when I understand a bit more